server/controllers/socket.controller.ts

```diff
@@ -5,13 +5,31 @@ import {
 	WebSocketEvent,
 	RoomUpdateEvent,
 	ConnectEvent,
+	OptionSelectedEvent,
 } from "../types/socket.ts";
+import calculateConfidence from "../utils/calculateConfidence.ts";
 import connectToDb from "../utils/connectToDb.ts";
 import { lookupRoom } from "../utils/db.ts";
 
 const sockets = new Map<string, WebSocket>();
 export const { rooms, users } = await connectToDb();
 
+/**
+ * Sends updated roomData to all voters in room
+ * @param roomData
+ * @returns void
+ */
+const sendRoomData = (roomData: RoomSchema): void => {
```

Comment:
Should this also include sending the room update to the moderator?

Reply:
Good catch.  I was thinking that the moderator was also a voter, but I see that might not be the case.  Fixing this.

Reply:
Added the logic to update the moderator as well as the voters.